### PR TITLE
chore(select): remove inner search

### PR DESCRIPTION
**Pull request by SteliosKon**
## Description

Remove inner search from select component

### CODE REVIEW

File: src/components/FormElements/Select/constants.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 export const MAX_MENU_HEIGHT = 300;
 export const OUTER_PLACEHOLDER = "Select...";
```
Comment (ThrasyvoulosKafasis) [inline src/components/FormElements/Select/constants.ts:2]: Rename to just `PLACEHOLDER`

File: src/components/FormElements/Select/components/CustomValueContainer.tsx [hunk 2]
```diff
@@ -1,58 +0,0 @@
...
-  };
-
-  const customProps = {
-    isDisabled: selectProps.isDisabled,
-    getClassNames: props.getClassNames,
-    innerProps: Object.assign({}, props.innerProps, { "data-role": "menuList" }),
-  };
-
-  return (
-    <ValueContainer {...props} selectProps={selectProps} css={CustomContainerStyles}>
```
Comment (ThrasyvoulosKafasis) [inline src/components/FormElements/Select/components/CustomValueContainer.tsx:36]: You should remove also `CustomContainerStyles` from `src/components/FormElements/Select/styles.ts`

**Review by ThrasyvoulosKafasis — CHANGES_REQUESTED** [review]

File: src/components/FormElements/Select/components/CustomSingleValue.tsx [hunk 3]
```diff
@@ -1,32 +0,0 @@
-import React, { FC, memo, useEffect, useRef, useState } from "react";
-import { components, SingleValueProps } from "react-select";
-import { CustomOption } from "../types";
-import Tooltip from "../../../Tooltip/Tooltip";
-
-const CustomSingleValue: FC<SingleValueProps<CustomOption>> = ({ children, ...props }) => {
```
[the remark this answers is not in the document]
Comment (SteliosKon) [reply src/components/FormElements/Select/components/CustomSingleValue.tsx:6]: We don't need it anymore

File: src/components/FormElements/Select/Select.stories.tsx [hunk 4]
```diff
@@ -152,58 +110,54 @@ withValueCreationValidation.args = {
...
   type: "creatable",
   isInputValid: (value: string): boolean =>
     /^(?=.*[^\d])(?=.*\S).+$/.test(value) &&
     !defaultOptions.find((option) => option.label === value),
-  checkIfInputIsSelected: (inputValue: string): string => {
-    return defaultOptions.find((option) => option.label === inputValue)
-      ? "Already selected"
-      : "No options";
-  },
+  // checkIfInputIsSelected: (inputValue: string): string => {
```
[the remark this answers is not in the document]
Comment (SteliosKon) [reply src/components/FormElements/Select/Select.stories.tsx:117]: I need to run this code by @lem8n

**Review by ThrasyvoulosKafasis — APPROVED** [review]

**Pull request merged**